libr/core/cmd_anal.c

```diff
@@ -61,12 +61,13 @@ static void type_cmd(RCore *core, const char *input) {
 	RListIter *it;
 	ut64 seek;
 	r_cons_break_push (NULL, NULL);
+	bool io_cache;
 	switch (*input) {
 	case 'a': // "afta"
 		seek = core->offset;
 		r_core_cmd0 (core, "aei");
 		r_core_cmd0 (core, "aeim");
-		bool io_cache = r_config_get_i (core->config, "io.cache");
+		io_cache = r_config_get_i (core->config, "io.cache");
```

Comment:
initialize io_cache in the same place you are defining it, no need to read the value more than once.. even if its not used in all the cases, its cleaner and the compiler can optimize it if it feels the need